### PR TITLE
chore: Updated release script to open-source

### PR DESCRIPTION
**Pull request by thomaspink**

### CODE REVIEW

**Comment by thomaspink** [comment]
Blocked until CI is up and running

**Comment by lukasholzer** [comment]
There is a good sample repo how to target the different release targets and know how to run the releasing:

https://github.com/labs42io/circleci-monorepo

but we said for now we will release manually and not from the ci.

File: tools/release/src/release-check.ts [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+      );
+    }
+  }
+  return false;
+}
+
+/** Entry-point for the release version determination script. */
+if (require.main === module) {
+  console.log(
+    shouldRelease(join(__dirname, '../../../'), 'dynatrace-oss', 'barista'),
```
[the remark this answers is not in the document]
Comment (thomaspink) [reply tools/release/src/release-check.ts]: why?

File: tools/release/src/release-check.ts [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+      red(
+        `The specified directory is not referring to a project directory. ` +
+          `There must be a ${italic(
+            'package.json',
+          )} file in the project directory.`,
+      ),
+    );
+    process.exit(1);
+  } else {
+    const packageJson = JSON.parse(readFileSync(packageJsonPath, 'utf-8'));
```
[the remark this answers is not in the document]
Comment (thomaspink) [reply tools/release/src/release-check.ts]: package json has to be a valid json. so no need for that

**Comment by thomaspink** [comment]
closed in favour of #403

**Pull request closed without merging**